flake.nix

```diff
@@ -0,0 +1,25 @@
+{
+  description = "A very basic flake";
+
+  outputs = { self, nixpkgs }: {
```

Comment:
There are no inputs in this flake. This means it will use your impure local registry, `nixpkgs` by default, much like channels. You should add `inputs.nixpkgs.url = "github:nixos/nixpkgs/nixos-21.11";` to avoid that.

Reply:
Thanks!